### ISSUE TITLE
Solid color in properties not working as it should

### ISSUE THREAD

**Issue by philipzae** [M7S/xfce4-dockbarx-plugin#4]
If I set the panel plugin to have a solid color and set the alpha value other an 100%, the blending between the solid color and the background color of the panel isn't correct.
![image](https://user-images.githubusercontent.com/4412114/87241027-edc77400-c42f-11ea-8db3-32cec9c0011b.png)

**Comment by xuzhen** [comment]
The "Solid color" option sets the background color independently. It has nothing to do with the panel's background color.

In addition, it also has a bug and may not really work in the GTK3 version.

**Comment by philipzae** [comment]
Yes I understand 'solid color' sets the background color independent from the panel's background color, but the color should be blended with the background color when the alpha slider is adjusted, instead its bringing an independent color which isn't the panel's background color.
